packages/sihl-email/sihl-email.0.1.6/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "A Sihl service for sending emails using Lwt"
+description: """
+
+A Sihl service for sending emails using Lwt. Various email transports are provided that can be used in production or testing such as SMTP, Sendgrid, in-memory and console printing."""
+maintainer: ["josef@oxidizing.io"]
+authors: ["Josef Erben" "Aron Erben" "Miko Nieminen"]
+license: "MIT"
+homepage: "https://github.com/oxidizing/sihl"
+doc: "https://oxidizing.github.io/sihl/"
+bug-reports: "https://github.com/oxidizing/sihl/issues"
+depends: [
+  "dune" {>= "2.7" & >= "2.7"}
```

Comment:
It does no harm but is there a way to avoid this duplication here?

Reply:
I pushed a fix. Do you have an idea why dune could be generating that from the `dune-project` file?

Reply:
I think dune uses `(lang dune 2.7)` to infer the minimal dune version required and appends it to the bounds you added. But this is just a guess, probably @rgrinberg knows better

Reply:
It's a bug. Feel free to report